cytominer_database/commands/command_ingest_variable_engine.py

```diff
@@ -0,0 +1,90 @@
+import click
```

Comment:
just confirming - did we run `black` on this file? Line 81 makes me think no

Reply:
I double-checked and re-ran black; it only reformatted `setup.py` and `doc/conf.py` - i.e. `cytominer_database/commands/command_ingest_variable_engine.py` seems to have been formatted already ?

Reply:
<img width="706" alt="Screenshot 2020-07-29 at 16 33 11" src="https://user-images.githubusercontent.com/27351364/88814273-29956400-d1ba-11ea-86ae-9b4d61558151.png">